extensions/amp-form/0.1/form-submit-service.js

```diff
@@ -19,7 +19,7 @@ import {Observable} from '../../../src/observable';
 
 /**
  * @typedef {{
- *   form: HTMLFormElement,
+ *   form: !HTMLFormElement,
```

Comment:
why this change?

Reply:
The change is because the `formSubmitService.js#fire` function can only be triggered by a form that exists, so the type should be non-nullable. It also allows consumers to use the `FormSubmitEventDef#form` property without checking if it's non-null first.

Reply:
Good call, thanks for tidying this up.